src/controllers/index.ts

```diff
@@ -2,8 +2,10 @@ import 'app/controllers/CheckYourAppealController';
 import 'app/controllers/ConfirmationController';
 import 'app/controllers/EntryController'
 import 'app/controllers/EvidenceUploadController';
```

Comment:
At some point we might look in to consistent names. Either we decide to use evidence or file. Does not need to be this PR though.